### DISCUSSION TITLE
Add  HelmRelease - Chart version for prometheus & Grafana will improve visibility

### DISCUSSION THREAD

**Discussion by haimari** [fluxcd/flux2#1131] [Proposals]
Currently when using flux monitoring Stack, prometheus does not know the Helm Chart version of the`HelmRelease`.
It would be great to see the `version` of each HelmRelease (Helm Chart + Helm App version).

Also, For example Here (screenshot) a Developer can see that there is an issue, but there is no indication for the Helm chart version

![image](https://user-images.githubusercontent.com/11332517/111912371-f3bb2d80-8a71-11eb-996a-be8e1bebb5f0.png)

While it is possible to see the Chart + App version of Helm Charts with Octant, I think it would be more useful to View directly in Grafana, since most metrics are visible there anyway, and it does not require dedicated installation on the teams devices + adding a plugin for helm, etc...

![image](https://user-images.githubusercontent.com/11332517/111912558-df2b6500-8a72-11eb-91b8-e7d62b476ed4.png)

Is there an option to add this to prometheus so this could be queried in addition to this ?
`gotk_reconcile_condition{namespace=~"$namespace",type="Ready",status="False",kind=~"Kustomization|HelmRelease"}`

**Comment by stefanprodan** [comment]
You can use this exporter and have a table with all release versions https://github.com/sstarcher/helm-exporter

**Reply by Jimmy-Newtron** [reply]
Good to know but you are saying to add an additional element to be managed on the platform, while the proposal makes a lot of sense to have the information in the same dashboard as the deployment information
